purchase_requisition_auto_rfq/models/purchase_requisition.py

```diff
@@ -61,3 +70,43 @@ def _create_purchase_order(self, seller_id):
         po = self.env["purchase.order"].with_context(**ctx).create(vals)
         po._onchange_requisition_id()
         return po
+
+    def action_all_rfq_related_send(self):
+        self.ensure_one()
+        rfqs = self.env["purchase.order"].search([("requisition_id", "=", self.id)])
+        ir_model_data = self.env["ir.model.data"]
+        try:
+            template_id = ir_model_data.get_object_reference(
+                "purchase", "email_template_edi_purchase"
+            )[1]
+        except ValueError:
+            template_id = False
+        ctx = dict(
+            self.env.context,
+            default_model="purchase.order",
+            active_model="purchase.order",
+            active_id=rfqs.ids[0],
+            active_ids=rfqs.ids,
+            default_template_id=template_id,
+            default_composition_mode="mass_mail",
+        )
+        return {
+            "type": "ir.actions.act_window",
+            "view_mode": "form",
+            "res_model": "mail.compose.message",
+            "target": "new",
+            "context": ctx,
+        }
+
+
+class PurchaseRequisitionLine(models.Model):
+    _inherit = "purchase.requisition.line"
+
+    prod_has_supplier = fields.Boolean(
+        compute="_compute_prod_has_supplier", string="Product without suppliers"
+    )
+
+    @api.depends("product_id.product_tmpl_id.seller_ids")
+    def _compute_prod_has_supplier(self):
+        for rec in self:
+            rec.prod_has_supplier = bool(rec.product_id.product_tmpl_id.seller_ids)
```

Comment:
You can make this related too.
If you don't make the related field, change the string to "Product has suppliers".

```suggestion
    seller_ids = fields.One2many(
        comodel="product.supplierinfo",
        related="product_id.product_tmpl_id.seller_ids",
    )
```